src/styles/page.scss

```diff
@@ -0,0 +1,8 @@
+body {
+  margin: 0;
+}
+
+.page {
+  display: flex;
```

Comment:
you can add justify content and align item here so that there is no spaces between your blocks

Reply:
Those spaces are present because of the gap rule

Reply:
I'm actually using flexbox not position absolute

Reply:
Actually i meant to leave a comment in the mixin part. Apparently i couldn't leave it sorry for that.